resume.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title> Rossy </title>
+        	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap" rel="stylesheet">
+    		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      		<style>
+                  body {
+    padding-left: 10%;
+    padding-right: 10%;
+    margin-bottom: 10%;
+    background-color: #c3fbf4;
+}
+.Sticky {
+    width: 20vw;
+    height: 30vh;
```

Comment:
When I look at this selector using browser dev tools, I can see that the text is actually escaping your set height:

<img width="542" alt="Screen Shot 2020-10-04 at 3 24 16 PM" src="https://user-images.githubusercontent.com/2653980/95025142-c01c5080-0655-11eb-92c2-accbf06c9b4b.png">

This may be a place where you want to use a height value set in pixels and not relative to the viewport.